src/pages/dashboard.vue

```diff
@@ -357,7 +358,17 @@ const coaEffectiveDate = computed(() => {
           </div>
         </template>
         <div class="flex justify-center py-5">
-          <p>{{ $t('text.filing.completeYourFiling') }}</p>
+          <BcrosPartyInfo
+            v-if="bootstrapFiling?.filing?.incorporationApplication?.parties"
+            name="director"
+            :role-type="RoleTypeE.DIRECTOR"
+            :show-address="false"
+            :show-email="false"
+            :expand-top-item="true"
+          />
+          <p v-else>
+            {{ $t('text.filing.completeYourFiling') }}
+          </p>
```

Comment:
Can you please fix the alignment / whitespace for the current directors?

![image](https://github.com/user-attachments/assets/6828cea3-02a3-44ed-9786-9cf0d38a5246)

Reply:
I've moved it to the left now and gotten rid of a lot of the extra bottom space